packer/core.go

```diff
@@ -74,6 +74,7 @@ type BasicStore interface {
 
 type BuilderStore interface {
 	BasicStore
+	Has(name string) bool
```

Comment:
Seeing as these interfaces embed a BasicStore, which already requires a Has(string) bool method do we need to add it to this interface. I think we can safely drop it from these interfaces.

Reply:
Oh yeah if `BasicStore` already has it we can drop it for sure, good call